.goreleaser.yml

```diff
@@ -131,3 +131,51 @@ winget:
           owner: go-task
           name: winget-pkgs
           branch: "bump-task-to-{{.Tag}}"
+
+dockers:
+  - image_templates:
+      - "ghcr.io/go-task/task:v{{.Version}}-amd64"
+    use: buildx
+    dockerfile: Dockerfile
+    extra_files:
+      - completion/bash/task.bash
```

Comment:
Direct usage of these files is [deprecated](https://taskfile.dev/deprecations/completion-scripts/).

Reply:
Oh damn, I completely missed that. Will fix!

Reply:
Yeah unfortunately, they are still used in a few places. This is why we deprecated them instead of removing them. We need to slowly remove the references to them. We could probably still add completions to the image by writing a `.bashrc` during the build and using the new method

Reply:
I thought about generating the completions in the container, but then I decided against it.

It doesn't make much sense to generate the bash completions given the image is based on scratch IMHO, and even then, if we generate and include one set of completions we might as well generate all.

I think that, given the completions can be generated from the binary, it is just as simple to add a `RUN` command in your dockerfile to set up your required completions.

There are already instructions for generating the completions so that should have it covered. At the end of the day, the container image is a means to distribute the binary, right?

Let me know what you think!